src/vmshepherd/presets/abstract.py

```diff
@@ -69,7 +69,10 @@ def __init__(self, runtime, defaults):
         raise NotImplementedError
 
     def get_preset(self, preset_name):
-        return self._presets[preset_name]
+        try:
+            return self._presets[preset_name]
+        except KeyError:
+            raise Exception(f'Preset {preset_name} not found')
```

Comment:
In the next iteration we should introduce some common parent class for VmShepherd's exceptions